### PR TITLE
chore: bump zwave-js to 8.1.0

### PR DESCRIPTION
**Pull request by AlCalzone**
Updating `zwave-js` should solve a little DDOS problem we have with our telemetry infrastructure right now.

### CODE REVIEW

File: src/lib/controller/message_handler.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ export class ControllerMessageHandler {
     switch (message.command) {
       case ControllerCommand.beginInclusion: {
         const success = await driver.controller.beginInclusion(
-          message.includeNonSecure
+          message.includeNonSecure!
```
Comment (MartinHjelmare) [inline src/lib/controller/message_handler.ts:17]: Are these other changes needed?

Comment (AlCalzone) [reply src/lib/controller/message_handler.ts:17]: There is an `| undefined` missing from the function signature. You know what - I should probably do a 8.1.1 anyways.

File: tsconfig.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
     "types": ["node"],
     "esModuleInterop": true,
     "resolveJsonModule": true,
-    "strict": true
+    "strict": true,
+    "skipLibCheck": true
```
Comment (AlCalzone) [inline tsconfig.json:13]: This is necessary and I'd recommend it anyways.

**Comment by AlCalzone** [comment]
As discussed on discord, there should be no need for this. `zwave-js` is only a peer dependency.

**Review by raman325 — APPROVED** [review]

**Pull request closed without merging**